ecs-agent/api/ecs/statechange.go

```diff
@@ -124,6 +126,9 @@ func (c *ContainerStateChange) String() string {
 // String returns a human readable string representation of a TaskStateChange.
 func (change *TaskStateChange) String() string {
 	res := fmt.Sprintf("%s -> %s", change.TaskARN, change.Status.String())
+	if len(change.ClusterARN) != 0 {
+		res += fmt.Sprintf(", ClusterARN: %s", change.ClusterARN)
```

Comment:
nit(non-blocking): can we follow the format on L128 "%s -> %s" here?

Reply:
Sure. Will fix it if there is any blocking comment.